src/Hidden/features.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import {Meta} from '@storybook/react'
+import React from 'react'
+import {Hidden} from './Hidden'
+import {Box, Button} from '..'
+
+const meta: Meta = {
+  title: 'Drafts/Components/Hidden/Features',
+  component: Hidden,
+}
+export default meta
```

Comment:
```suggestion
export default {
  title: 'Drafts/Components/Hidden/Features',
  component: Hidden,
} as Meta
```

Reply:
Random, should we be using `ComponentMeta` and `ComponentMeta<typeof Hidden>` now in storybook? 🤔 Looking at the TS examples in: https://storybook.js.org/docs/react/api/csf

Reply:
This is a great point Josh! Feels like we should be moving onto `ComponentMeta` and `ComponentMeta<typeof Hidden>` .  Love this summary [here](https://github.com/storybookjs/storybook/discussions/16650#discussioncomment-1617480). I saw in multiple places we import the types explicity from the component. This way it should be cleaner 🙌🏻